lldb/source/Plugins/SymbolFile/DWARF/DebugNamesDWARFIndex.h

```diff
@@ -65,6 +65,10 @@ class DebugNamesDWARFIndex : public DWARFIndex {
   void GetFunctions(const RegularExpression &regex,
                     llvm::function_ref<bool(DWARFDIE die)> callback) override;
 
+  StatsDuration::Duration GetIndexTime() override {
+    return m_fallback.GetIndexTime();
```

Comment:
Can we get a test to exercise this case? From the description it sounds like it covers the case of partial debug_names. Is it possible to craft an example where all the time is spent in the fallback case? So that checking for a non-zero index time would be enough to show that it is working.